lib/logwriter.c

```diff
@@ -567,10 +567,9 @@ log_writer_emit_suppress_summary(LogWriter *self)
 
   msg_debug("Suppress timer elapsed, emitting suppression summary");
 
-  m = log_msg_new_empty();
-  m->timestamps[LM_TS_STAMP] = m->timestamps[LM_TS_RECVD];
+  m = log_msg_new_local();
   m->pri = self->last_msg->pri;
-  m->flags = LF_INTERNAL | LF_LOCAL;
+  m->flags |= LF_INTERNAL;
 
```

Comment:
Thanks! I replaced the last commit which modified logwriter with this one: https://github.com/balabit/syslog-ng/pull/1115/commits/02a8c072529816bfd7594f5365d9894bc629cb7b

The other commits are untouched by the force push.